GNUmakefile

```diff
@@ -150,7 +150,9 @@ cov:
 	gocov test $(GOFILES) | gocov-html > /tmp/coverage.html
 	open /tmp/coverage.html
 
-test: other-consul dev-build vet
+test: other-consul dev-build vet test-internal
+
+test-internal:
```

Comment:
This allows the `test-ci` target to retry only the go-test part not re-do all the vet and other deps too which is pointless.

I actually just spotted that we still do `go test -i` inside this target which is pointless to retry so I'll refactor that out too.